kubernetes/config/samples/Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM torchelastic/examples:0.1.0rc1
 
 USER root
 RUN apt update && apt install -y wget unzip
-RUN wget http://cs231n.stanford.edu/tiny-imagenet-200.zip &&  unzip tiny-imagenet-200.zip -d /data
\ No newline at end of file
+RUN wget http://cs231n.stanford.edu/tiny-imagenet-200.zip &&  unzip tiny-imagenet-200.zip -d /data
+ADD https://raw.githubusercontent.com/pytorch/elastic/master/examples/imagenet/main.py /opt
```

Comment:
since you are using `torchelastic/examples` as the base image, this should already be under `/workspace/examples/imagenet/main.py`. Are you not seeing this? is there a reason why it has to go under `/opt`?

Reply:
ah. just notice we copy everything here. https://github.com/pytorch/elastic/blob/master/Dockerfile#L5

This is great. I will make an update.